content/pages/privacy.nl.md

```diff
@@ -86,7 +89,9 @@ DWH wil je er tevens op wijzen dat je de mogelijkheid hebt om een klacht in te d
 
 DWH neemt de bescherming van jouw gegevens serieus en neemt passende maatregelen om misbruik, verlies, onbevoegde toegang, ongewenste openbaarmaking en ongeoorloofde wijziging tegen te gaan. Als jij het idee hebt dat jouw gegevens toch niet goed beveiligd zijn of er aanwijzingen zijn van misbruik, neem dan contact op via [secretaris@dwhdelft.nl](mailto:secretaris@dwhdelft.nl).
 
-De ledenadministratie wordt offline bijgehouden en is alleen toegankelijk voor de ledenadministrateur die onder verantwoordelijkheid van de secretaris handelt. De ledenadministrateur verstrekt op verzoek delen van de administratie aan bestuurs- en commissieleden indien noodzakelijk voor het functioneren van de vereniging. Hierbij kan gedacht worden aan het nalopen van de ledenlijst bij aanvang van de algemene ledenvergadering, of het versturen van e-mails aan leden. De ledenadministratie verstrekt enkel die informatie die nodig is voor het opgegeven doel; in bijna alle gevallen slechts een deel van de administratie. Leden waaraan gegevens verstrekt worden dienen een geheimhoudingsverklaring te hebben ondertekend.
+De ledenadministratie wordt online bijgehouden in een zelfgemaakt systeem ("ReMember") en is alleen toegankelijk voor het minimaal nodige aantal bestuurs- en commissieleden. Allen hebben hiervoor een geheimhoudingsverklaring getekend. De ledenadministrateur verstrekt op verzoek delen van de administratie aan bestuurs- en commissieleden indien noodzakelijk voor het functioneren van de vereniging. Hierbij kan gedacht worden aan het nalopen van de ledenlijst bij aanvang van de algemene ledenvergadering, of het versturen van e-mails aan leden. De ledenadministratie verstrekt enkel die informatie die nodig is voor het opgegeven doel; in bijna alle gevallen slechts een deel van de administratie. Leden waaraan gegevens verstrekt worden dienen een geheimhoudingsverklaring te hebben ondertekend.
+
+Deze administratie wordt enkel versleuteld overgedragen naar en opgeslagen op Europese servers van DigitalOcean. Zie [de website van DigitalOcean](https://www.digitalocean.com/legal/privacy-policy) voor meer informatie. Updates ten behoeve van de databeveiliging worden tijdig uitgevoerd.
```

Comment:
toch? of herinner ik m nu verkeerd

Reply:
Klopt op zich, zou het digitalocean verhaal weglaten en "Deze administratie wordt enkel versleuteld overgedragen naar en opgeslagen op Europese servers" toevoegen aan de vorige alinea